components/catalogs/front/src/components/CatalogEdit/hooks/useCatalogForm.ts

```diff
@@ -25,6 +26,13 @@ export const useCatalogForm = (id: string): Result => {
     const [dirty, setDirty] = useState<boolean>(false);
     const [errors, setErrors] = useState<CatalogFormErrors>([]);
     const saveCatalog = useSaveCatalog();
+    const {data: catalogErrors} = useCatalogErrors(id);
```

Comment:
```suggestion
    const {data: initialErrors} = useCatalogErrors(id);
```
what do you think of this naming? or maybe `existingErrors` ?

Reply:
ok for initialErrors, this makes senses.